### PR TITLE
To-Do-App Using React JS

### PR DESCRIPTION
**Pull request by for-ashraf**
In this PR I have performed all the steps defined in https://ibaslogic.com/react-tutorial-for-beginners/. Are mentioned below:
[Part 1 - React tutorial: The beginner's guide to learning React in 2020](https://ibaslogic.com/react-tutorial-for-beginners/).
[Part 2 - Working with React form and handling event](https://ibaslogic.com/react-form-handling/).
[Part 3 – How to implement CSS in Reactjs app](https://ibaslogic.com/css-in-reactjs-app/).
[Part 4 – How to edit to dos items](https://ibaslogic.com/how-to-edit-todos-items-in-react/).
[Part 5 – Persisting React state in local storage](https://ibaslogic.com/persisting-react-state-in-local-storage/) (Optional).
[Part 6 – Getting started with React lifecycle methods](https://ibaslogic.com/react-lifecycle-methods/).
[Part 7 – Getting started with React hooks](https://ibaslogic.com/react-hooks-tutorial/).
[Part 8 – How to use SVG icons in React](https://ibaslogic.com/how-to-use-svg-icons-in-react-project/) (Optional).
[Part 9 – Routing with React Router](https://ibaslogic.com/routing-with-react-router/).
[Part 10 – How to add hamburger menu in React](https://ibaslogic.com/how-to-add-hamburger-menu-in-react/) (Optional).
[Part 11 – Deploying React app to GitHub Pages](https://ibaslogic.com/deploying-react-app-to-github-pages/) (Optional).

### CODE REVIEW

File: README.old.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# to-do-using-react-js
```
Comment (lucas-crodrigues) [inline README.old.md]: [OPTIONAL] it might be a good idea to delete this file, just so as to keep the repo cleaner

File: README.md [hunk 2]
```diff
@@ -1,2 +1,69 @@
-# to-do-using-react-js
+# To-Do App Using React js
```
Comment (lucas-crodrigues) [inline README.md:1]: Here in your README.md file, you're missing some important information, like how to set up the repo.

[OPTIONAL] You can add all the other sections given at [this template](https://github.com/microverseinc/readme-template) for a professional README file

**Review by lucas-crodrigues — CHANGES_REQUESTED** [review]
Hi @for-ashraf, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] All linters are passing :heavy_check_mark: 
- [ ] No bugs :heavy_check_mark: 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review for more details :point_down: :point_down: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @lucas-crodrigues in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (for-ashraf) [reply README.md:1]: Dear, I have incorporated all the changes in the repository. You are requested to review the code again, please.

**Review by lucas-crodrigues — APPROVED** [review]
# Status: Approved :tada: :tada: :tada: 
Hi @for-ashraf    :wave: :wave: 

200w  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**